server/src/api/v1.py

```diff
@@ -196,11 +222,15 @@ def job_builder(data: dict, auth_token: str):
 
     priority_level = data.get("job_priority", 0)
     job_queue = data["job_queue"]
+    reserve_data = data.get("reserve_data", {})
+    # default reservation timeout is 1 hour
+    reservation_timeout = reserve_data.get("timeout", 3600)
```

Comment:
I think there's a corresponding line in the device connector as well. Is that still necessary?

Reply:
I think so - this value isn't passed to the device connector unless explicitly specified in the job